pindel/workflow.cwl

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "pindel workflow"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+inputs:
+    reference:
+        type: File
+        secondaryFiles: [".fai", "^.dict"]
```

Comment:
Same comment about picard sequence dictionary, is it used by the pindel workflow?